static/bidder-params/criteo.json

```diff
@@ -1,30 +1,50 @@
-{
-    "$schema": "http://json-schema.org/draft-04/schema#",
-    "title": "Criteo adapter params",
-    "description": "The schema to validate Criteo specific params accepted by Criteo adapter",
-    "type": "object",
-    "properties": {
-        "zoneid": {
-            "type": "number",
-            "description": "Impression's zone ID.",
-            "minimum": 0
-        },
-        "networkid": {
-            "type": "number",
-            "description": "Impression's network ID.",
-            "minimum": 0
-        }
-    },
-    "anyOf": [
-        {
-            "required": [
-                "zoneid"
-            ]
-        },
-        {
-            "required": [
-                "networkid"
-            ]
-        }
-    ]
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Criteo adapter params",
+    "description": "The schema to validate Criteo specific params accepted by Criteo adapter",
+    "type": "object",
+    "properties": {
+        "zoneid": {
+            "type": "number",
+            "description": "Impression's zone ID.",
+            "minimum": 0
+        },
+        "zoneId": {
+            "type": "number",
+            "description": "Impression's zone ID, preferred.",
+            "minimum": 0
+        },
+        "networkid": {
+            "type": "number",
+            "description": "Impression's network ID.",
+            "minimum": 0
+        },
+        "networkId": {
+            "type": "number",

```

Comment:
Last change request. Could you please change the types from "number" to "integer"? Number includes floating point values, but the underlying data structure is an int64 - so I think integer would be most appropriate. This is confusing some automated config systems.

Reply:
Indeed that makes sense
I'm pushing the change

Reply:
Excellent. Thank you. We'll merge this in for today's release.